File: sdkwork-ai-sdk-java/src/main/java/com/sdkwork/ai/api/I18nResourceApi.java
package com.sdkwork.ai.api;

import com.sdkwork.ai.http.HttpClient;
import com.sdkwork.ai.model.*;
import java.util.List;
import java.util.Map;

public class I18nResourceApi {
    private final HttpClient client;
    
    public I18nResourceApi(HttpClient client) {
        this.client = client;
    }

    public PlusApiResultListLabelOption getOptions(Map<String, Object> params) throws Exception {
        return (PlusApiResultListLabelOption) client.get(ApiPaths.aiPath("/v1/open/i18n/options"), params);
    }

    public PlusApiResultListLabelOption createOptions(Map<String, Object> params) throws Exception {
        return (PlusApiResultListLabelOption) client.post(ApiPaths.aiPath("/v1/open/i18n/options"), null, params);
    }
}

File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/api/I18nResourceApi.kt
package com.sdkwork.ai.api

import com.sdkwork.ai.*
import com.sdkwork.ai.http.HttpClient

class I18nResourceApi(private val client: HttpClient) {

    suspend fun getOptions(params: Map<String, Any>? = null): PlusApiResultListLabelOption? {
        return client.get(ApiPaths.aiPath("/v1/open/i18n/options"), params) as? PlusApiResultListLabelOption
    }

    suspend fun createOptions(params: Map<String, Any>? = null): PlusApiResultListLabelOption? {
        return client.post(ApiPaths.aiPath("/v1/open/i18n/options"), null, params) as? PlusApiResultListLabelOption
    }
}
